### DISCUSSION TITLE
Create an edge from source handle to target handle of the same node.

### DISCUSSION THREAD

**Discussion by kaushalendra-pandey** [xyflow/xyflow#2378] [General]
I am trying to create an edge from the source handle to target handle of the same node. The edge is getting created but the issue is it is not visible as it is behind the node. Any idea on how we can solve this?

**Comment by bcakmakoglu** [comment]
Lucky for you, @moklick just recently posted [this custom Edge](https://codesandbox.io/s/sharp-orla-vyjqen?file=/src/CustomEdge.tsx) that connects on the same node.

**Reply by kaushalendra-pandey** [reply]
Works very well. Thanks @bcakmakoglu

**Reply by royhuntley** [reply]
How can i change this to be a smoothstep edge, rather than Bezier ? THanks.
